tools/dump_all_tables_data_only.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
@driusan @xlecours 
simplest format of a script dumping only the data from each table with all necessary wrapping
 - truncation
 - disable foreign key checks
 - no drop table
 - no create table
 - column names specified

obviously multiple hardcoded values need to be removed here and probably passed as arguments instead but this is just an example